modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/criterion/DLItemSelectorCriterionCreationMenuRestrictionUtil.java

```diff
@@ -64,8 +62,20 @@ public static Set<String> getAllowedCreationMenuUIItemKeys(
 		return allowedCreationMenuUIItemKeys;
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
+	private static final PropertyServiceReferenceMapper
+		<String, DLItemSelectorCriterionCreationMenuRestriction>
+			_propertyServiceReferenceMapper =
+				new PropertyServiceReferenceMapper<>("model.class.name");
```

Comment:
@GabrielHenriquedev @rafaprax Please inline this mapper, no need to make it an attribute of this class